renovate.json

```diff
@@ -8,5 +8,11 @@
     "smoke-tests"  
   ],
   "automerge": true,
-  "masterIssue": true
+  "masterIssue": true,
+  "packageRules": [
+    {
+      "packageNames": ["eslint"],
+      "rangeStrategy": "bump"
```

Comment:
for documentation purposes: https://docs.renovatebot.com/configuration-options/#rangestrategy
(and historical, for looking up why this change was made)

thanks a bunch for doing this! <3